<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorWindow</class>
 <widget class="QMainWindow" name="CalculatorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>590</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>590</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
      background-color: #1a305c;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: transparent;
    padding: 15px;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="display">
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::RightToLeft</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPlainTextEdit#display {
    padding: 15px 20px;
    background-color: #1a202c;
    color: #f7fafc;
    font-size: 32px;
    font-weight: bold;
    border: none;
    selection-background-color: #4299e1;
}

QScrollBar:horizontal {
    height: 12px;
    background: #4a5568;
    margin: 0 20px 2px 20px;
    border-radius: 6px;
    border: 1px solid #2d3748;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QGridLayout" name="numbersLayout">
          <item row="2" column="1">
           <widget class="QPushButton" name="button5">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
 background-color: #4a5568;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>5</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="buttonDecimal">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
  background-color: #718096;
    font-size: 20px;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>.</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="buttonLeftParenthesis">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
  background-color: #718096;
    font-size: 20px;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>(</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="buttonDelete">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
    background-color: #e53e3e;
    font-weight: bold;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>âŒ«</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="button1">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
 background-color: #4a5568;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="button6">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
 background-color: #4a5568;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>6</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="buttonClear">
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <bold>true</bold>
              <underline>false</underline>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
    background-color: #e53e3e;
    font-weight: bold;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>C</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="button3">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
 background-color: #4a5568;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="button2">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
 background-color: #4a5568;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="button4">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
 background-color: #4a5568;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}
</string>
            </property>
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="button0">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
 background-color: #4a5568;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QPushButton" name="buttonRightParenthesis">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
  background-color: #718096;
    font-size: 20px;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="button9">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
 background-color: #4a5568;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>9</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="button7">
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <fontweight>DemiBold</fontweight>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
 background-color: #4a5568;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>7</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="button8">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
 background-color: #4a5568;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>8</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="operationsLayout">
          <item>
           <widget class="QPushButton" name="buttonDivide">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
  background-color: #3182ce;
    font-size: 20px;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>/</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonMultiply">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
  background-color: #3182ce;
    font-size: 20px;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>*</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonMinus">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
  background-color: #3182ce;
    font-size: 20px;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPlus">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
  background-color: #3182ce;
    font-size: 20px;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonEquals">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
  background-color: #38a169;
    font-size: 20px;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>=</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="functionLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="buttonSqrt">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
   background-color: #6b46c1;
    font-size: 16px;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>âˆš</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="buttonReciprocal">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
  background-color: #6b46c1;
    font-size: 16px;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>1/x</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="buttonSin">
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <fontweight>DemiBold</fontweight>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
    background-color: #6b46c1;
    font-size: 16px;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>sin</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="buttonCos">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
 background-color: #6b46c1;
    font-size: 16px;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>cos</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="buttonTan">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
 background-color: #6b46c1;
    font-size: 16px;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>tg</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="buttonCot">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    border: none;
    border-radius: 8px;
    color: #f7fafc;
    font-size: 18px;
    font-weight: 600;
    padding: 15px;
    margin: 4px;
    min-width: 50px;
    min-height: 50px;
 background-color: #6b46c1;
    font-size: 16px;
}

QPushButton:pressed {
    background-color: #ebf4ff;
    border-color: #a0aec0;
}

QPushButton:hover {
    border: 2px solid #a0aec0;
}</string>
            </property>
            <property name="text">
             <string>ctg</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
